File: Examples/Scripts/CodeGeneration/CodeGen61/Project1.pas
pas : StereotypeStereotypeProducer <<@meta>> <<UMLCategory>> <<Project>> Project1
 pas : StereotypeStereotypeProducer <<@meta>> <<UMLCategory>> <<Library>> Library1
 ; // Library1
 pas : StereotypeStereotypeProducer <<@meta>> <<UMLCategory>> <<Library>> Library2
 ; // Library2
 pas : StereotypeStereotypeProducer <<@meta>> <<UMLCategory>> <<Library>> Library3
 ; // Library3
 pas : StereotypeStereotypeProducer <<@meta>> <<UMLCategory>> <<Program>> Program1
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Class>> Class1
  ; // Class1
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Class>> Class2
   pas : OPERATOR NamedWordProducer %INHERITS %G
   ; // %G
   pas : OPERATOR NamedWordProducer %IMPLEMENTS %R
   ; // %R
  ; // Class2
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Class>> Class3
  ; // Class3
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Class>> Class4
   pas : OPERATOR NamedWordProducer %INHERITS %G
   ; // %G
  ; // Class4
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Interface>> Interface1
  ; // Interface1
  pas : StereotypeStereotypeProducer <<@meta>> <<UMLClass>> <<Interface>> Interface2
  ; // Interface2
 ; // Program1
; // Project1

File: Examples/Scripts/CodeGeneration/CodeGen61/Project1.cpp
cpp : <<Project>> Project1
 cpp : <<Library>> Library1
 ; // Library1
 cpp : <<Library>> Library2
 ; // Library2
 cpp : <<Library>> Library3
 ; // Library3
 cpp : <<Program>> Program1
  cpp : <<Class>> Class1
  ; // Class1
  cpp : <<Class>> Class2
   cpp : %INHERITS %G
   ; // %G
   cpp : %IMPLEMENTS %R
   ; // %R
  ; // Class2
  cpp : <<Class>> Class3
  ; // Class3
  cpp : <<Class>> Class4
   cpp : %INHERITS %G
   ; // %G
  ; // Class4
  cpp : <<Interface>> Interface1
  ; // Interface1
  cpp : <<Interface>> Interface2
  ; // Interface2
 ; // Program1
; // Project1
